public/Copy-DbaDatabase.ps1

```diff
@@ -164,6 +164,9 @@ function Copy-DbaDatabase {
     .LINK
         https://dbatools.io/Copy-DbaDatabase
 
+	.INPUT
```

Comment:
Correct keyword to use `.INPUTS`

Reply:
Ref: https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comment_based_help?view=powershell-7.3#inputs

The proper description should include the full type name and list of commands that are supported (see ref above). We don't commonly put that in the command docs because we support multiple types.

Current description would make it sound like only Get-DbaDatabase can be used.

Reply:
As well, in my view `InputObject` parameter description already covers this detail. Having to maintain `.INPUTS` and `.OUTPUTS` (they should go together) is hard across all of our commands because so many types are allowed as inputs.

Reply:
Ups, I was on the wrong track - please ignore my comment.

Reply:
I don't see `.INPUTOBJECT` [listed here](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comment_based_help?view=powershell-7.3).  What does it mean?  Maybe that's not complete documentation?

Reply:
We generally use `.PARAMETER InputObject` to document what type of objects can be piped to the command. We currently only use `.INPUTS` in five public and one private command in the module, but always together with `.OUTPUTS`.
So I'm ok with adding `.INPUTS` and `.OUTPUTS` here, but please have a look at the other usages to align the woring. In general, I would rather see that as a bigger change done by a maintainer throughout the module.